common/src/main/java/us/myles/ViaVersion/util/CollectionsUtil.java

```diff
@@ -0,0 +1,48 @@
+package us.myles.ViaVersion.util;
+
+import lombok.experimental.UtilityClass;
+import us.myles.ViaVersion.api.Via;
+
+import java.lang.reflect.Constructor;
+import java.lang.reflect.InvocationTargetException;
+import java.util.HashMap;
+import java.util.Map;
+
+@UtilityClass
+public class CollectionsUtil {
+    private static Constructor<?> fastUtilLongObjectHashMap;
+    private static Constructor<?> fastUtilIntObjectHashMap;
+
+    static {
+        try {
+            fastUtilLongObjectHashMap = Class.forName("it.unimi.dsi.fastutil.longs.Long2ObjectOpenHashMap").getConstructor();
```

Comment:
This could be improved by trying to use "org.bukkit.craftbukkit.libs.it.unimi.fastutil" package

Reply:
Objection; We can’t rely on that. See bungeecord

Reply:
on 1.14.4 its doesn't exists
![image](https://user-images.githubusercontent.com/18630111/69914435-62c3db80-1455-11ea-850b-595f0cc4ef89.png)

Reply:
You can try using both packages

Reply:
@Xernium it's fallbacking to java.util

Reply:
is it good enough? 
```java
    public static Class<?> getClass(String className) throws ClassNotFoundException {
        try {
            return Class.forName("org.bukkit.craftbukkit.libs."+className);
        } catch (ClassNotFoundException ignored) {
        }
        return Class.forName(className);
    }